paparazzi/src/main/java/app/cash/paparazzi/Paparazzi.kt

```diff
@@ -254,13 +262,13 @@ class Paparazzi(
 
             var image = bridgeRenderSession.image
             renderExtensions.forEach {
-              image = it.render(snapshot, view, image)
+              image = it.postRender(view, image)
```

Comment:
discussed offline.  would be nice to also render the view bound color rects using view hierarchy manipulation vs post-image rendering.